problems/symmetric_substrings.rb

```diff
@@ -1,8 +1,10 @@
 class String
-  # Write a String#symmetric_substrings method that returns an array of substrings
-  # that are palindromes, e.g. "cool".symmetric_substrings => ["oo"]
+  # Define a method `symmetric_substrings` that returns 
```

Comment:
Should be `String#symmetric_substrings` or otherwise specify that it's being monkey-patched onto the String class